stock_picking_compute_delivery_date/model/picking.py

```diff
@@ -216,7 +232,11 @@ def compute_all_delivery_dates(self, cr, uid, use_new_cursor=True,
                     'Could not update delivery date for product %s',
                     product.name)
 
+        # Publish messages on pickings
+        self.picking_notify(cr, uid, messages, context=context)
```

Comment:
If picking notify fails and `use_new_cursor` is True, `cr` won't be rollbacked nor closed. Should be in a try: except: